src/main/java/woocommerce/entity/product/Product.java

```diff
@@ -20,9 +20,9 @@
     private String slug;
     @JsonElement(key = "permalink")
     private String permalink;
-    @JsonElement(key = "date_created")
+    @JsonElement(key = "date_created", type = TypeOfJsonElement.ISO8601_DATETIME)
```

Comment:
It's good that we define this clear, but should we add one more property like "format" or something, so we would have 1 Date type and multiple formats, that would be more convenient than remembering the names.